actionpack/lib/action_controller/metal/flash.rb

```diff
@@ -37,10 +37,15 @@ def add_flash_types(*types)
             request.flash[type]
           end
           helper_method(type) if respond_to?(:helper_method)
-
           self._flash_types += [type]
         end
       end
+
+      # Extend AbstractController::Base::action_methods to remove any added flash types
+      # from the set
```

Comment:
I think this method is simple enough that we don't need internal documentation for it.

```suggestion
```